Atomic/syscontainers.py

```diff
@@ -398,12 +434,18 @@ def _get_ostree_repo(self):
 
     def update_system_container(self, name):
         self.args.display = False
+        self.args.remote = None
 
         repo = self._get_ostree_repo()
         if not repo:
             raise ValueError("Cannot find a configured OSTree repo")
 
         path = os.path.join(self._get_system_checkout_path(), name)
+        with open(os.path.join(path, "info"), 'r') as info_file:
+            info = json.loads(info_file.read())
+            self.args.remote = info['remote']
+            if self.args.remote:
+                util.write_out("Updating a container with a remote rootfs. Only changes to config will be applied.")
```

Comment:
I think it is safer this way, so that we can control when the container is updated, we should just improve the way versions are managed somehow and keep track of them in the container info.

For containers with an already exploded rootfs, updating the configuration file to point to another version is an update, so I am still wondering if we could drop this warning as it looks to me that we are just recreating the config file (i.e. just set new values if specified via `--set`) but no update is really taking place.